map-sdk/angular/angular-basic-example/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "angular-basic-example",
+  "version": "0.0.0",
+  "scripts": {
```

Comment:
We should probably add MIT license field in package.json. And an MIT `LICENSE` file in root. @isaacbrodsky does that sound good?